### ISSUE TITLE
no-unused-declaration does not caught unused interface

### ISSUE THREAD

**Issue by lukewlms** [cartant/tslint-etc#17]
**Expected:** tslint-etc catches unused items that will cause tsc to fail if --noUnusedLocals

**Actual:** an unused interface is not caught by tslint-etc, but causes tsc build to fail with --noUnusedLocals 

**Sample code:**

(The const is caught by tslint-etc no-unused-declaration, but the interface is not.)

```ts
interface IProgressScreenState {
  daysInWindow: number | null;
}

const foo = 1;
```

---

Thanks for taking a look!

**Comment by cartant** [comment]
Should be fixed in 1.5.1.
